sdk/storage/azure_storage_blob/tests/blob_client.rs

```diff
@@ -15,19 +20,41 @@ async fn test_get_blob_properties() -> Result<(), Box<dyn Error>> {
     let credential = DefaultAzureCredentialBuilder::default().build()?;
 
     // Act
+    let container_client = BlobContainerClient::new(
+        &endpoint,
+        String::from("testcontainer1"),
```

Comment:
We need to figure out how to seed the container names similar to other languages (seeded random based on test name) since we keep hitting conflicting test container names.

Reply:
For now we will assign these tests all unique container names.